staging/storage/redis/redis-controller.yaml

```diff
@@ -10,19 +10,18 @@ spec:
     metadata:
       labels:
         name: redis
+        role: master
     spec:
       containers:
       - name: redis
         image: gcr.io/google_containers/redis:v1
         ports:
-        - containerPort: 6379
+          - containerPort: 6379
```

Comment:
I know this will work, but some YAML parsers actually fail to parse this. Please revert this one.